docs/en/index.md

```diff
@@ -0,0 +1,60 @@
+# Kotlin in Telegram
+
+[In Russian](https://kotlinby.github.io/kotlin-telegram/)
+
+## Chats
+
+* [@kotlin_lang](https://t.me/kotlin_lang) — Discussion of Kotlin language, IDE, general questions. Kotlin/Native
+* [@kotlin_mobile](https://t.me/kotlin_mobile) — Kotlin-related questions on Android development
+* [@kotlin_jvm](https://t.me/kotlin_jvm) — Kotlin on JVM — Server-side (Spring Boot, Ktor, etc) & Desktop (TornadoFX)
+* [@kotlin_js](https://t.me/kotlin_js) — Questions on Kotlin/JS
+* [@kotlin_start](https://t.me/kotlin_start) — Chat for novice Kotlin developers
+* [@kotlin_meta](https://t.me/kotlin_meta) — Organizational questions, discussion of chat moderation
+
+## Channels
+
+* [@kotlin_jobs](https://t.me/kotlin_jobs) — Kotlin openings
+* [@TheDailyKotlin](https://t.me/TheDailyKotlin) — Kotlin news
+
+## Bots
+
+* [@tryktbot](https://t.me/tryktbot) — Try Kotlin Bot
+* [@tgkotbot](https://t.me/tgkotbot) — Kotlin Bot (Moderation and Statistics)
+
+## How to ask questions
+
+Some points which would help you to compose the question to maximize the chance of getting an answer:
+
+1. You should ask yourself if your question is related to Kotlin. If the only reason to ask here is the fact you are solving your problem in Kotlin, and it would still sound meaningful for Java, there is a high chance that you should write to different channel related to your topic.
```

Comment:
> you should write to different channel

This is perfectly correct in context of Slack, but in Telegram channels have different meaning.

Reply:
What is your proposal? Replace channel with group?

Reply:
Yep, it is 'Group' according to Telegram UI, or 'Chat'.